server/Cargo.lock

```diff
@@ -3429,9 +3429,9 @@ dependencies = [
 
 [[package]]
 name = "rustls"
-version = "0.21.10"
+version = "0.21.11"
```

Comment:
I think we should bump the major version as well. Will do that if similar change goes smoothly downstream.

Reply:
n/m, we don't even have a direct dependency here :sweat_smile: